homeassistant/components/motion_blinds/strings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "config": {
+    "flow_title": "Motion Blinds",
+    "step": {
+      "user": {
+        "title": "Motion Blinds",
+        "description": "You will need the 16 character API Key, see https://www.home-assistant.io/integrations/motion_blinds/#retrieving-the-key for instructions",
+        "data": {
+          "host": "IP address",
+          "api_key": "API key"
+        }
+      }
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
+      "already_in_progress": "Config flow for this Motion gateway is already in progress",
+      "connection_error": "Failed to connect, please try again"
```

Comment:
I think that "already_in_progress" and "connection_error" support translations from homeassistant/strings.json:
"[%key:common::config_flow::abort::already_in_progress%]"
"[%key:common::config_flow::error::cannot_connect%]"

Reply:
thanks, just pushed a commit!